recipes/pykep/meta.yaml

```diff
@@ -0,0 +1,47 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "pykep" %}
+{% set version = "2.0" %}
+{% set sha256 = "274a2053e5d85a6d5c9f60aaeeb2f70505de3b94fb5c9b06e68f92f3defaf86d" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/esa/pykep/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - boost 1.65.1
+
+  run:
+    - boost 1.65.1
+
+
+test:
+  commands:
+    - test -f
```

Comment:
Can you add a test for an import from the python package? 
```yaml
test:
   imports:
      - pykep
```

Reply:
sure